### DISCUSSION TITLE
What do the dimensions of the state returned by the environment mean? What are the corresponding parameters in the paper?

### DISCUSSION THREAD

**Discussion by 945716994** [godka/Pensieve-PPO#22] [General]
比如说state[0, -1]是last quality，对应论文中的哪个参数？是xt还是什么？
![image](https://user-images.githubusercontent.com/36669447/156908274-2f0a61bb-ed49-42e0-b8a6-ee117378c6ce.png)

**Comment by godka** [comment]
See here.
https://github.com/godka/Pensieve-PPO/blob/7e795b25c1e739e299d328da7115aebc1074bcd7/src/env.py#L93

PS: please create an issue in **English**.
